packages/node_modules/@webex/react-component-activity-item-base/src/index.js

```diff
@@ -95,6 +95,7 @@ function ActivityItemBase({
           color="black-100"
           name={isFlagged ? 'flag-active_16' : 'flag_16'}
           onClick={handleOnFlag}
+          aria-pressed={isFlagged}
```

Comment:
This is so that when the flag is active it announces 'selected', as per Applause requirements.